backend/src/routes/index.js

```diff
@@ -2,11 +2,13 @@ const express = require('express');
 
 const { authMiddleware, getAuthRoutes } = require('./auth.js');
 const { getSongRoutes } = require('./songs.js');
+const { getSpotifyRoutes } = require('./spotify');
 
 function getRoutes() {
   const router = express.Router();
   router.use('/auth', getAuthRoutes());
   router.use('/songs', authMiddleware, getSongRoutes());
+  router.use('/spotify', getSpotifyRoutes());
```

Comment:
Could you add the auth middleware to the spotify route? We wouldn't want people linking spotify accounts to nothing...